src/data/hooks/useApy.ts

```diff
@@ -1,50 +1,64 @@
 import { useQuery } from "@tanstack/react-query"
-import { getApy as getApy_AAVE_V2_STABLE_CELLAR } from "data/actions/CELLAR_V0815/getApy"
 import { CellarNameKey, ConfigProps } from "data/types"
-import { CellarStakingV0815, CellarV0815 } from "src/abi/types"
+import { useGet10DaysShareValueQuery } from "generated/subgraph"
+import { getPrevious10Days } from "utils/calculateTime"
 import { useCreateContracts } from "./useCreateContracts"
-import { useSommelierPrice } from "./useSommelierPrice"
+
+const previous10Days = getPrevious10Days()
 
 export const useApy = (config: ConfigProps) => {
   const { cellarContract, stakerContract } =
     useCreateContracts(config)
 
-  const sommPrice = useSommelierPrice()
+  const [{ fetching, data, error }, reexecute10Days] =
+    useGet10DaysShareValueQuery({
+      variables: {
+        epoch: previous10Days,
+        cellarAddress: config.id,
+      },
+    })
+
+  const latestData = data?.cellar?.dayDatas.at(-1)
+  const prevDayLatestData = data?.cellar?.dayDatas.at(-2)
 
   const AAVE_V2_STABLE_CELLAR_QUERY_ENABLED = Boolean(
     config.cellarNameKey === CellarNameKey.AAVE &&
       cellarContract.provider &&
-      stakerContract?.provider
+      stakerContract?.provider &&
+      latestData &&
+      prevDayLatestData
   )
 
   const query = useQuery(
-    ["USE_APY", sommPrice.data, config.cellar.address],
+    ["USE_APY", config.cellar.address],
     async () => {
-      if (!sommPrice.data) {
-        throw new Error("Sommelier price is undefined")
-      }
       if (config.cellarNameKey === CellarNameKey.AAVE) {
-        return await getApy_AAVE_V2_STABLE_CELLAR(
-          cellarContract as CellarV0815,
-          stakerContract as CellarStakingV0815,
-          sommPrice.data
-        )
+        const yieldGain =
+          (Number(latestData?.shareValue) -
+            Number(prevDayLatestData?.shareValue)) /
+          Number(prevDayLatestData?.shareValue)
+        const apy = yieldGain * 52 * 100
+        return {
+          apyLabel: "Expected APY",
```

Comment:
Expect APY is the sum of the Cellar APY (< cellar apy >) and the Rewards APY (< potentialStakingApy >).